contents/blog/cookieless-analytics.md

```diff
@@ -0,0 +1,41 @@
+---
+date: 2021-08-13
+title: Cookieless Analytics - How To Track Website and App Users Without Third-Party Cookies
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: marcus-hyett
+featuredImage: ../images/blog/filename.png
+featuredImageType: full
+---
+
+Google has stated that they intend to deprecate the use of 3rd party cookies in 2023. But why is this important, and how will this affect product analytics?
+
+## What are 3rd party cookies?
+
+Cookies are small files stored on the user’s machine, helping you identify a repeat user to your product or website. This makes things like staying logged in over multiple days possible, but it also makes it possible to track whether the person who followed an ad yesterday came back today to purchase something.
+3rd party cookies are a particular flavour of cookie provided by an external service and not your website directly. This external service can be a tool or marketing product that you work with to optimize ads.
+
+## What happens when third-party cookies go away?
+
+When 3rd party cookies are deprecated, any external tools you use in your product or website to track users - either for analytics or marketing purposes - will cease to work correctly.
```

Comment:
I'd push for the most simplified/direct language everywhere where possible - e.g. 'will cease to work correctly' -> 'will stop working correctly.'